integration/build/build_test.go

```diff
@@ -168,7 +168,11 @@ func TestBuildMultiStageParentConfig(t *testing.T) {
 	image, _, err := apiclient.ImageInspectWithRaw(ctx, "build1")
 	assert.NilError(t, err)
 
-	assert.Check(t, is.Equal("/foo/sub2", image.Config.WorkingDir))
+	expected := "/foo/sub2"
+	if testEnv.DaemonInfo.OSType == "windows" {
+		expected = `C:\foo\sub2`
```

Comment:
If we have more situations like this, we could consider having an `is.EqualPath()` helper